backend/benefit/helsinkibenefit/settings.py

```diff
@@ -346,7 +345,6 @@
 AZURE_ACCOUNT_NAME = env("AZURE_ACCOUNT_NAME")
 AZURE_ACCOUNT_KEY = env("AZURE_ACCOUNT_KEY")
 AZURE_CONTAINER = env("AZURE_CONTAINER")
-AZURE_URL_EXPIRATION_SECS = env("AZURE_URL_EXPIRATION_SECS")  # default 900s
```

Comment:
Is this dead code removal? AFAIK you are removing `AZURE_URL_EXPIRATION_SECS` completely from the whole code base.